Mesh4all/m4a-firmware
# git: show stats betweem releases

### Contribution description
It's a simple but useful stats generator between releases, it's taken from `RIOT-OS`; https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/release-stats/release-stats.sh and works with few changes

Check out  [the following Wiki section for more info.](https://github.com/RIOT-OS/RIOT/wiki/release-statistics) 

### Testing procedure

Go to `dist/tools/release-stats` and execute the script `release-stats.sh`

Note than, 
- a fresh clone or fork is required if we need to track the latest tags or releases
- it's should be optimized after the merge

### Issues/PRs references
None
